feconf.py

```diff
@@ -198,6 +199,10 @@
 # StateAnswersModel.submitted_answer_list schema change.
 CURRENT_STATE_ANSWERS_SCHEMA_VERSION = 1
 
+# This value should be updated if the schema of LearnerAnswerInfo
+# dict schema changes.
+CURRENT_LEARNER_ANSWERS_INFO_SCHEMA_VERSION = 1
```

Comment:
Rename this to `CURRENT_LEARNER_ANSWER_INFO_SCHEMA_VERSION`, since the field name is `learner_answer_info_schema_version`.

Reply:
Done